tests/console/validate_lvm_raid1.pm

```diff
@@ -83,27 +81,22 @@ sub _reboot {
 }
 
 sub _check_raid_disks_after_reboot {
-    my ($config, $actual_num_devs) = @_;
+    my ($config, $expected_num_devs) = @_;
     record_info('get state after reboot');
     select_console 'root-console';
 
     assert_script_run 'mdadm --detail ' . $config->{raid1}->{name};
     my $active_devs = script_output("mdadm --detail " . $config->{raid1}->{name} . " |grep \"Active Devices\" |awk '{ print \$4 }'");
-    # if the number of disk in raid1 list are not the same report a soft failure
-    if ($active_devs != $config->{actual_num_devs}) {
-        record_soft_failure 'bsc#1150370 - disk is restored during the booting' if is_sle('>=15');
-    }
-    else {
-        assert_equals($actual_num_devs, $active_devs, "Active devices should be still " . $config->{expected_num_devs});
-    }
+    # if the deactivated disk is not reactivated after reboot, run _restore_raid_disk
```

Comment:
if no failure now, at least some `record_info` to spot among job which one has it or not has it. wdyt?

Reply:
Yes, good idea. I also removed some lines that were repeating, before and after _restore_raid_disk is called.